mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -133,7 +135,7 @@ def __init__(self, name=None, bus=None, use_settings=True):
         self.gui = SkillGUI(self)
 
         self._bus = None
-        self._enclosure = None
+        self._enclosure = EnclosureAPI()
```

Comment:
This change seems out of scope for this PR

Reply:
enclosure was created in self.bind with bus and skill id, but that is not needed.

with this PR we use set_bus and set_id like in other objects for consistency. so its as good a place as any to initialize the object